webapp/channels/src/components/integrations/abstract_incoming_hook.test.tsx

```diff
@@ -80,10 +82,9 @@ describe('components/integrations/AbstractIncomingWebhook', () => {
     });
 
     test('should match snapshot, displays client error when no initial hook', () => {
-        const newInitialHook = {};
-        const props = {...requiredProps, initialHook: newInitialHook};
+        const props = {...requiredProps};
+        delete props.initialHook;
```

Comment:
@M-ZubairAhmed because `initialHook` is now optional rather than required and also allowing empty objects, this needed to be updated to allow stripping `initialHook` out for the test. I've used delete but other options would be restructuring how the mocked props are created.

Reply:
this is fine as well. thank you for fixing it